web-tester/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log (web-tester)
 
+## [10.0.0] - 2022-10-06
```

Comment:
maybe quickly check if I've merged some other PRs from dependabot...

Reply:
for all 3 artifacts

Reply:
I have :)
![image](https://user-images.githubusercontent.com/42733123/194276036-bdec9c0b-0c07-4a45-9791-9b196a9c8122.png)